modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -42,98 +76,149 @@ const PolicyDetail = () => {
 				/>
 			</div>
 
-			{amountOfPolicyDetail.map(
-				(curentPolicyDetail, indexPolicyDetail) => (
-					<div className="bg-neutral-0 p-6" key={indexPolicyDetail}>
-						{active === navBarLabels.Vehicles && (
-							<>
-								<div className="d-flex flex-row flex-wrap justify-content-between mb-3">
-									{indexPolicyDetail !== 0 && (
-										<div className="align-self-start col-12 layout-line mb-6 mt-1"></div>
-									)}
-
-									<div className="align-self-start w-50">
-										<h5>2016 Ford Edge</h5>
-
-										<img
-											className="policy-detail-img w-75"
-											src={`${getWebDavUrl()}/ford-edge_icon.svg`}
-										/>
-									</div>
+			{active === navBarLabels.Vehicles &&
+				applicationData?.vehicleInfo?.form.map(
+					(curentVehicle: PolicyDetails, indexVehicle: number) => (
+						<div
+							className="bg-neutral-0 pl-6 policy-detail-border pr-6 pt-6"
+							key={indexVehicle}
+						>
+							<div className="d-flex flex-row flex-wrap justify-content-between">
+								{indexVehicle !== 0 && (
```

Comment:
Please, change it.
```suggestion
								{indexVehicle && (
```

Reply:
indexVehicle and indexDriver is not boolean value, it's a number. If I check this way the numeric value is rerendered as one of the columns of the content. Any other suggestions for this case?